scripts/exregional_get_obs_ccpa.sh

```diff
@@ -79,7 +79,7 @@ while [[ ${current_fcst} -le ${fcst_length} ]]; do
   vdd=`echo ${vdate} | cut -c7-8`    # day (DD) of valid time
   vhh=`echo ${vdate} | cut -c9-10`       # forecast hour (HH)
 
-  vhh_noZero=$(expr ${vhh} + 0)
+  vhh_noZero=$( printf "%d" "${vhh#0}" )
```

Comment:
@michelleharrold Is the `#0` at the end something I suggested in our meeting?  If so, sorry, it's not correct!  Scrap that and change to `$((10#${vhh}))` to force `${vhh}` to be interpreted as a decimal number.

Reply:
@gsketefian -- what you originally suggested worked...until I tested on forecast hour 08 and 09, and then I got the dreaded octet error. So I had to modify it. I will go ahead and change to what you suggested and run it!